src/site/content/en/blog/lighthouse-ci/index.md

```diff
@@ -0,0 +1,351 @@
+---
+title: "Performance Monitoring with Lighthouse CI"
+authors:
+  - katiehempenius
+date: 2020-07-22
+description: |
+  Learn how to setup Lighthouse CI and integrate it into developer workflows.
+hero: hero.png
```

Comment:
How about something like this for the hero?

![image](https://user-images.githubusercontent.com/4713486/88338947-08ca9b80-ccee-11ea-9ba2-3374c521c3b7.png)

I feel like something that takes up most of the hero space when you first open the page will be more visually appealing. Reminder that the hero should be 3200 pixels by 960 pixels.